src/apiCalls.js

```diff
@@ -1,8 +1,9 @@
 //IMPORTS 
 import { getRandomUser } from "./users";
-import { pageLoadRenders } from "./domUpdates";
+import { pageLoadRenders, hideSpinner } from "./domUpdates";
 import { copyItem } from "./helper-functions";
 import { populateTags, calculateRecipeCost, getIngredientAmounts, getInstructions } from './recipes';
+import { config } from "../config.js"
```

Comment:
pls comment this out before merging, otherwise looks amazing

Reply:
done